### PR TITLE
tools: chagen scanregions endkey default limit

### PR DESCRIPTION
**Pull request by IcePigZDB**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: doing
- Other reference link(s):
    * directly code PR: tikv/pd/pull/4674
    * tracking issue: tikv/pd/issues/4400

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-control.md [hunk 1]
```diff
@@ -878,7 +878,7 @@ Encoding 格式示例：
 
 ### `region keys [--format=raw|encode|hex] <start_key> <end_key> <limit>`
 
-用于查询某个 key 范围内的所有 Region。支持不带 `endKey` 的范围。`limit` 的默认值是 16，不带 `endKey` 时的默认值是 `-1`（表示没有限制)。示例如下：
+用于查询某个 key 范围内的所有 Region。支持不带 `endKey` 的范围。`limit` 的默认值是 `16`，设为`-1`则无数量限制。示例如下：
```
Comment (TomShawn) [inline pd-control.md:881]:
```suggestion
用于查询某个 key 范围内的所有 Region。支持不带 `endKey` 的范围。`limit` 的默认值是 `16`，设为 `-1` 则表示无数量限制。示例如下：
```

File: pd-control.md [hunk 2]
```diff
@@ -895,42 +895,42 @@ Encoding 格式示例：
 }
 ```
 
-显示从 a 开始的所有 Region 信息，数量上限为 20：
+显示 [a, z) 范围内的所有 Region 信息，数量上限为16：
```
Comment (TomShawn) [inline pd-control.md:898]:
```suggestion
显示 [a, z) 范围内的所有 Region 信息，数量上限为 16：
```

**Review by TomShawn — APPROVED** [review]
Rest LGTM

**Review by nolouch — APPROVED** [review]

**Pull request merged**